expression/builtin.go

```diff
@@ -226,6 +239,17 @@ func (b *baseBuiltinFunc) evalDuration(row []types.Datum) (types.Duration, bool,
 	return val.GetMysqlDuration(), false, errors.Trace(err)
 }
 
+func (b *baseBuiltinFunc) evalJSON(row []types.Datum) (json.JSON, bool, error) {
```

Comment:
Yeah, but we may use an individual PR to change this.
We'd better change CreateJSON to return *json.JSON,
and then change evalJSON to return *json.JSON